src/datachain/lib/dc.py

```diff
@@ -859,6 +859,17 @@ def mutate(self, **kwargs) -> "Self":
                 dist=cosine_distance(embedding_text, embedding_image)
         )
         ```
+
+        This method also has enhanced capabilities for renaming the columns. If column
+        with the current name exists then its renamed. Orelse a new column is created
```

Comment:
`Orelse` - I'm not sure this is an existing word in English. Can we rephrase it a bit please. E.g. use `Otherwise`. Wdyt?